Qt/mainwindow.cpp

```diff
@@ -495,20 +535,31 @@ void MainWindow::createMenus()
 		->addEventChecked(&g_Config.bAutoRun);
 
 	// Debug
-	MenuTree* debugMenu = new MenuTree(this, menuBar(),   QT_TR_NOOP("De&bug"));
-	debugMenu->add(new MenuAction(this, SLOT(lmapAct()),      QT_TR_NOOP("Load Map File...")))
+	MenuTree* debugMenu = new MenuTree(this, menuBar(),   QT_TR_NOOP("&Debug"));
+	debugMenu->add(new MenuAction(this, SLOT(breakonloadAct()),   QT_TR_NOOP("Break on load")))
+		->addEventChecked(&g_Config.bAutoRun);
```

Comment:
This is inversed - `g_Config.bAutoRun == true` means it should be unchecked.

-[Unknown]